include/geode/inspector/topology/brep_topology.h

```diff
@@ -39,6 +39,13 @@ namespace geode
 
 namespace geode
 {
+    struct opengeode_inspector_inspector_api BRepInspectionResult
```

Comment:
BRepTopologyInspectionResult? This is not the complete result for the BRep inspection, still missing BRep meshes inspection